EnglishFilter/sections/foreign.txt

```diff
@@ -3208,6 +3208,10 @@ bhaskar.com#%#var canABP = true;
 !---------------------------------------------------------------------
 ! https://github.com/AdguardTeam/AdguardFilters/issues/46399
 sosok.grid.id#?#.container > h2.title__default > span:contains(/^PROMOTED CONTENT$/)
+! https://github.com/AdguardTeam/AdguardFilters/issues/46349
+103.194.171.75,idtube.me##.overlay.bm
```

Comment:
cannot open the site. Is not it bookmark button?

Reply:
I was able to access this site with Singapore IP

Reply:
@Alex-302  I didn't find class .bm in any other elements
![screenshot](https://user-images.githubusercontent.com/18536328/71471981-4f045e80-27e2-11ea-83b6-704116acf598.png)

Reply:
This is not an ad. Also check another rules.